sql/migrations/20220721212432_world.sql

```diff
@@ -0,0 +1,35 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20220721212432');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20220721212432');
+-- Add your query below.
+
+-- SFK: Arugal intro script
+DELETE FROM `generic_scripts` WHERE `id`=10000;
+INSERT INTO `generic_scripts` (`id`, `delay`, `priority`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(10000, 0, 0, 4, 46, 134217728, 1, 0, 0, 0, 0, 4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'SFK Intro - Arugal Invisible'),
+(10000, 5, 0, 4, 46, 134217728, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'SFK Intro - Arugal Visible'),
```

Comment:
Why are you toggling UNIT_FLAG_AURAS_VISIBLE ???
That does not make units invisible. It's for showing auras to hostile players. This flag is applied when you are afflicted by a mage's Detect Magic debuff.